lib/index.js

```diff
@@ -35,16 +37,88 @@ function getMetaData(command, root) {
   });
 }
 
-function getDependencies(command, root, targetFile, allowMissing, args) {
+// Hack:
+// We're using Zeit assets feature in order to support Python and Go testing 
+// within a binary release. By doing "path.join(__dirname, 'PATH'), Zeit adds
+// PATH file auto to the assets. Sadly, Zeit doesn't support (as far as I
+// understand) adding a full folder as an asset, and this is why we're adding
+// the required files this way. In addition, Zeit doesn't support 
+// path.resolve(), and this is why I'm using path.join()
+function createAssets(){
+  assets = [];
+  assets.push(path.join(__dirname, '../plug/pip_resolve.py'));
+  assets.push(path.join(__dirname, '../plug/distPackage.py'));
+  assets.push(path.join(__dirname, '../plug/package.py'));
+  assets.push(path.join(__dirname, '../plug/reqPackage.py'));
+  assets.push(path.join(__dirname, '../plug/utils.py'));
+
+  assets.push(path.join(__dirname, '../plug/requirements/fragment.py'));
+  assets.push(path.join(__dirname, '../plug/requirements/parser.py'));
+  assets.push(path.join(__dirname, '../plug/requirements/requirement.py'));
+  assets.push(path.join(__dirname, '../plug/requirements/vcs.py'));
+  assets.push(path.join(__dirname, '../plug/requirements/__init__.py'));
+
+  return assets;
```

Comment:
Maybe `plug` is not such a good name, considering that the go plugin assets will also be dumped in a temp folder?